Writerside/topics/Client.md

```diff
@@ -1,72 +0,0 @@
-# Clients
```

Comment:
Why did you remove this one?

Reply:
It only had some parts that are already mentioned in Clients and projects